aidial_adapter_bedrock/llm/consumer.py

```diff
@@ -27,7 +27,7 @@ def append_content(self, content: str):
         pass
 
     @abstractmethod
-    def close_content(self):
+    def close_content(self, finish_reason: FinishReason = FinishReason.STOP):
```

Comment:
This is confusing: there is one default in the base class (Consumer, STOP) and another default in the subclass (ChoiceConsumer, None). It seems like the abstract class doesn't have to have any parameter defaults, because in it couldn't be instantiated anyway.

Reply:
Made them consistent.